src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        ArrayList<Integer> arr = new ArrayList<>();
+        arr.add(3);
+        arr.add(7);
+        arr.add(null);
+        arr.add(5);
+        arr.add(2, 2);
+        System.out.println(arr.get(0));
+        System.out.println(arr.get(1));
+        System.out.println(arr.get(2));
+        System.out.println(arr.get(3));
+        System.out.println(arr.get(4));
+    }
+}
```

Comment:
For this, you do not need to create the Main class, use the ready-made ArrayListTest class
```suggestion
```